front/pages/api/stripe/webhook.ts

```diff
@@ -601,14 +588,20 @@ async function handler(
                 status: "ended",
                 endDate: new Date(),
               });
-              const workspaceId = matchingSubscription.workspace.sId;
-              const auth = await Authenticator.internalAdminForWorkspace(
-                workspaceId
-              );
-              await revokeUsersForDowngrade(auth);
-              await archiveConnectedAgents(auth);
-              await deleteConnectedDatasources(auth);
-              await checkStaticDatasourcesSize(auth);
+
+              const scheduleScrubRes =
+                await launchScheduleWorkspaceScrubWorkflow({
+                  workspaceId: matchingSubscription.workspace.sId,
+                });
+              if (scheduleScrubRes.isErr()) {
+                return apiError(req, res, {
```

Comment:
would panic flag it (we need to know and fix)

Reply:
👌